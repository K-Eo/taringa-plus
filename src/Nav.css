.nav-head {
  height: 70px;
  padding-top: 11px;
}

.logo {
  background: url(./assets/images/logo.png) no-repeat;
  display: block;
  height: 48px;
  width: 270px;
}

.logo-fallback {
  display: none;
}

.main-nav {
  border-radius: 4px 4px 0 0;
  border: none;
  margin-bottom: 0;
  min-height: 0;
}

.main-nav > .navbar-nav > li > a {
  padding: 8px 15px;
  font-size: 14px;
  font-weight: bold;
  color: #333;
}

.main-nav > .navbar-nav > li > a:hover {
  padding: 8px 15px;
  font-size: 14px;
  font-weight: bold;
  color: #555;
}

.main-nav > .navbar-nav > li.active > a,
.main-nav > .navbar-nav > li.active > a:hover,
.main-nav > .navbar-nav > li.active > a:active {
  background: #333;
  color: #fff;
  text-shadow: #000 0 1px 0;
  box-shadow: inset 0px 2px 11px 1px rgba(0,0,0,0.5);
}

.main-nav > .navbar-nav > li:first-child > a {
  border-radius: 4px 0 0 0;
}

.main-nav > .navbar-nav.navbar-right > li:last-child > a {
  border-radius: 0 4px 0 0;
}

.secondary-nav {
  background: #007394;
  box-shadow: inset 0px 2px 5px 0px rgba(0,0,0,0.5);
  font-size: 12px;
  font-weight: bold;
  padding: 6px 5px 0 10px;
  border-bottom: none;
}

.secondary-nav > li {
  margin-right: 10px;
  margin-bottom: -2px;
}

.secondary-nav > li > a {
  border-radius: 0;
  line-height: 1;
  padding: 5px 15px;
  background: #3b8bac;
  color: #fff;
  border: none;
}

.secondary-nav > li > a:hover,
.secondary-nav > li > a:active {
  background: #3b8bac;
  text-decoration: underline;
}

.secondary-nav > li.active > a {
  color: #333;
}

.secondary-nav > li.active > a:hover {
  text-decoration: none;
}
