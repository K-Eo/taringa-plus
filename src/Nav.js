import React from 'react'
import { Menu, Container } from 'semantic-ui-react'

const Nav = (props) => {
  return (
    <Menu fixed='top' size='large'>
      <Container>
        <Menu.Item header>Taringa Plus!</Menu.Item>
      </Container>
    </Menu>
  )
}

export default Nav
