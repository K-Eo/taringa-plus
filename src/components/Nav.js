import React from 'react'
import { Menu, Container } from 'semantic-ui-react'
import { Link } from 'react-router-dom'

const Nav = (props) => {
  return (
    <Menu fixed='top' size='large'>
      <Container>
        <Menu.Item header as={Link} to='/'>Taringa Plus!</Menu.Item>
      </Container>
    </Menu>
  )
}

export default Nav
