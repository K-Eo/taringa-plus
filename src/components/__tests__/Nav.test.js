import React from 'react'
import Nav from '../Nav'
import renderer from 'react-test-renderer'
import { StaticRouter } from 'react-router'

it('Render nav', () => {
  const component = renderer.create(
    <StaticRouter location='/' context={{}}>
      <Nav/>
    </StaticRouter>
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
