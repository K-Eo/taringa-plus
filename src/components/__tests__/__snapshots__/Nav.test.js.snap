// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render nav 1`] = `
<div
  className="ui large top fixed menu"
>
  <div
    className="ui container"
  >
    <a
      className="header item"
      href="/"
      onClick={[Function]}
    >
      Taringa Plus!
    </a>
  </div>
</div>
`;
